client/src/routes/screens/Sessions/Sessions.tsx

```diff
@@ -0,0 +1,140 @@
+import {useNavigation} from '@react-navigation/native';
+import {NativeStackNavigationProp} from '@react-navigation/native-stack';
+import hexToRgba from 'hex-to-rgba';
+import React, {useCallback, useMemo} from 'react';
+import {useTranslation} from 'react-i18next';
+import {ListRenderItem} from 'react-native';
+import {FlatList} from 'react-native-gesture-handler';
+import LinearGradient from 'react-native-linear-gradient';
+import styled from 'styled-components/native';
+
+import {COLORS} from '../../../../../shared/src/constants/colors';
+import ExerciseWalletCard from '../../../lib/components/Cards/WalletCards/ExerciseWalletCard';
+import Gutters from '../../../lib/components/Gutters/Gutters';
+import MiniProfile from '../../../lib/components/MiniProfile/MiniProfile';
+import Screen from '../../../lib/components/Screen/Screen';
+import {
+  Spacer16,
+  Spacer60,
+  Spacer8,
+  TopSafeArea,
+} from '../../../lib/components/Spacers/Spacer';
+import TopBar from '../../../lib/components/TopBar/TopBar';
+import {Heading16} from '../../../lib/components/Typography/Heading/Heading';
+import {SPACINGS} from '../../../lib/constants/spacings';
+import useExerciseById from '../../../lib/content/hooks/useExerciseById';
+import useExerciseIds from '../../../lib/content/hooks/useExerciseIds';
+import {
+  ModalStackProps,
+  OverlayStackProps,
+} from '../../../lib/navigation/constants/routes';
+import {formatExerciseName} from '../../../lib/utils/string';
+
+const BottomGradient = styled(LinearGradient)({
+  position: 'absolute',
+  left: 0,
+  right: 0,
+  bottom: 0,
+  height: 40,
+});
```

Comment:
I see that we've missed this in `Journey` tab too. Thinking about creating a reusable component or add it to the `Screen` component. But I can do that in a separate branch.